drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -942,15 +941,16 @@ function DragAndDropBlock(runtime, element, configuration) {
         }
     };
 
-    var initializeSlider = function() {
-        if (pageLoaded) {
-            itemSlider = $root.find('.item-bank').bxSlider({
-               pager: false,
-                infiniteLoop: false,
-                startSlide: currentSlideIndex,
-                speed: 0
-            });
-        }
+    var sliderInitializer = function() {
```

Comment:
@murad-hubib The tests are breaking because your commits have overridden @jawad-khan 's `initializeSlider` method, and so the slider isn't getting created.

Can you please fix this?